src/shared/common/components/__tests__/workstack.spec.jsx

```diff
@@ -63,6 +69,8 @@ describe('Workstack component', () => {
         columns: [
             { displayName: 'Reference', dataAdapter: null, renderer: 'caseLink', dataValueKey: 'caseReference', isFilterable: true },
             { displayName: 'Full Name', dataAdapter: null, renderer: null, dataValueKey: 'fullName', isFilterable: true },
+            { displayName: 'Fullname', dataAdapter: 'primaryCorrespondent', renderer: null, dataValueKey: 'fullname', isFilterable: true },
```

Comment:
Slightly confused by this. "Full Name" is the column header. Do we need Fullname ?